scripts/deploy/__init__.py

```diff
@@ -130,9 +113,52 @@ def run_deploy_all(chain: str) -> None:
             address_provider.update_address(id, address_provider_inputs[id])
 
     # update metaregistry
-    update_metaregistry(chain, metaregistry, address_provider)
-
-    # TODO: transfer ownership to dao
+    update_metaregistry(chain_settings, metaregistry, address_provider)
+
+    # transfer ownership to the dao
+    owner = chain_settings.dao.ownership_admin
+
+    # addressprovider
+    current_owner = address_provider._storage.admin.get()
```

Comment:
better do `.admin()` everywhere

Reply:
can we do this in a separate PR? i'm not sure if this breaks backend